Gir.toml

```diff
@@ -2349,6 +2349,10 @@ manual_traits = ["WidgetExtManual"]
         [[object.function.parameter]]
         name = "allocation"
         const = true
+    [[object.function]]
+    name = "hide_on_delete"
+    ignore = true
```

Comment:
Please add a comment saying why the function is ignored (we don't do it enough...).